### ISSUE TITLE
FluidProperties record contains cp, lambda, that are often not used and are set to zero

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#316]
**Reported by otter on 16 Mar 2010 13:07 UTC**
Many pressure loss functions use the record:

```
Modelica.Fluid.Dissipation.Utilities.Records.General.FluidProperties
```

This record consists of cp, eta, lambda, rho. In most of the pressure loss functions, cp and lambda are not used and are set to zero. In these cases a record without cp and lambda should be used.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/316

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 16 Mar 2010 13:15 UTC**
Fix with 458abdd76e9bec360d904a53080f8da861223b73. A new record:

```
Modelica.Fluid.Dissipation.Utilities.Records.General.PressureLoss
```
 
introduced that has only variables rho and eta. Replaced the record at all places where it makes sense.

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 6 Oct 2010 08:31 UTC**
